pkgs/development/python-modules/whenever/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, fetchFromGitHub
+, pythonOlder
+, buildPythonPackage
+, poetry-core
+, backports-zoneinfo
+, tzdata
+, pytestCheckHook
+, pytest-mypy-plugins
+, hypothesis
+, freezegun
+}:
+
+buildPythonPackage rec {
+  pname = "whenever";
+  version = "0.3.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8.1";
```

```suggestion
  disabled = pythonOlder "3.8";
```
we usually do not bother which patch versions and this is likely a overeagerly contraint from upstream anyway.